### DISCUSSION TITLE
debian 9 Internal Server Error and cant install om ubuntu 20

### DISCUSSION THREAD

**Discussion by lultala** [modoboa/modoboa#2354] [Installer]
I use auto install with command sudo python3 run.py
all installed, but when i try enter to site i have error: Internal Server Error
I try install on ubuntu 20 (clear system) but install stoping on stage modoboa, i try start install with debug mode (screenshot)
![изображение](https://user-images.githubusercontent.com/52524303/132943198-c70436f0-6677-451c-9ff8-a5c24f26124f.png)
after hours waiting stoping on this step
![изображение](https://user-images.githubusercontent.com/52524303/132943206-fa765bff-85da-4ac2-a5fd-a5744d495a58.png)

**Comment by tonioo** [comment]
See https://github.com/modoboa/modoboa-installer/issues/388
